src/status_im/contexts/preview/quo/list_items/approval_info.cljs

```diff
@@ -0,0 +1,77 @@
+(ns status-im.contexts.preview.quo.list-items.approval-info
+  (:require
+    [quo.core :as quo]
+    [quo.foundations.resources :as resources]
+    [react-native.core :as rn]
+    [status-im.common.resources :as common.resources]
+    [status-im.contexts.preview.quo.preview :as preview]))
+
+(def descriptor
+  [{:type    :select
+    :key     :type
+    :options [{:key :spending-cap}
+              {:key :token-contract}
+              {:key :account}
+              {:key :spending-contract}
+              {:key :network}
+              {:key :date-signed}
+              {:key :collectible}
+              {:key :collectible-contract}
+              {:key :address}
+              {:key :community}]}
+   {:type :text
+    :key  :label}
+   {:type :text
+    :key  :description}
+   {:type :boolean
+    :key  :blur?}
+   {:type :boolean
+    :key  :unlimited-icon?}
+   {:type    :select
+    :key     :option-icon
+    :options [{:key   nil
+               :value "None"}
+              {:key :i/options}
+              {:key :i/chevron-right}]}])
+
+(defn- get-avatar-props
+  [type]
+  (cond
+    (#{:spending-cap :token-contract :network}
+     type)                         {:image (common.resources/get-mock-image :status-logo)}
+    (= type :collectible)          {:image (common.resources/get-mock-image :collectible2)}
+    (= type :account)              {:customization-color :orange
+                                    :emoji               "😇"}
+    (= type :collectible-contract) {:image (common.resources/get-mock-image :bored-ape)}
+    (= type :spending-contract)    {:network-image (resources/get-network :ethereum)
+                                    :image         (resources/get-dapp :coingecko)}
+    (= type :date-signed)          {:icon :i/signature}
+    (= type :address)              {:customization-color :blue
+                                    :full-name           "0 x"}
+    (= type :community)            {:image (common.resources/get-mock-image :status-logo)
+                                    :size  :size-32}))
+
+(defn view
+  []
+  (let [[state set-state] (rn/use-state {:type            :spending-cap
+                                         :label           "Label"
+                                         :description     "Description"
+                                         :blur?           false
+                                         :unlimited-icon? false
+                                         :option-icon     :i/options})]
+    [preview/preview-container
+     {:state                 state
+      :set-state             set-state
+      :blur?                 (:blur? state)
+      :show-blur-background? true
+      :blur-dark-only?       true
+      :descriptor            descriptor}
+     [quo/approval-info
+      (assoc state
+             :button-label    "Edit"
```

Comment:
The preview becomes more powerful/useful if we parameterize as much as we can, so the person can check in the UI how the component behaves in more scenarios. `tag-label` and `button-label` could be in the state.